### PR TITLE
Add Python 3.8 for Linux to Travis.

### PR DESCRIPTION
**Pull request by cmarqu**

### CODE REVIEW

**Comment by eric-wieser** [comment]
Perhaps we should remove the windows 3.7 testing, since we were only testing the latest version on windows before.

**Comment by themperek** [comment]
At the moment on windows it will use latest version that is distributed with conda so adding 3.8 will not make any difference. We can change this but maybe with sperate PR.

File: .travis.yml [hunk 1]
```diff
@@ -82,6 +90,15 @@ jobs:
       script:
         - docker run cocotb powershell 'python setup.py install ; make test'
 
+    - stage: SimTests
+      python: 3.8
```
Comment (imphil) [inline .travis.yml]: That's not having any effect, if you want to use a different Python version, you need to do so in the Dockerfile.windows. 

I'd suggest removing the Windows part for now, Linux Python 3.8 testing is already an important step forward.

Comment (cmarqu) [reply .travis.yml]: Okay, Linux only it is now.

**Pull request merged**